docker-compose.yaml

```diff
@@ -130,6 +134,10 @@ services:
       - 5050
     ports:
       - "4004:4004"
+    environment:
+      - http_proxy=${http_proxy}
+      - https_proxy=${https_proxy}
+      - no_proxy=${no_proxy},validator,eth0
```

Comment:
Can you explain what's going on here?

Reply:
I was using validator during one of dev work, eth0 shall be removed from it.